### PR TITLE
Bleeding, Dismemberment, and weapons rework. Oh my!

### PR DESCRIPTION
**Pull request by yackemflam**
Dismemberment is now its own variable, stop tying shit together people
Added bleeding to pointy weapons
Rapiers now makes people bleed a lot rather than I CUT YOUR FUCKING HEAD OFF WITH A THIN BLADE memes.

What this will fix.
w_class is a terrible thing to base things off of since everyone ignores it's actual use
Dismemberment can be used on shitty force weapons and have more !!FUN!! for everyone.
Bleeding is now actually a scary thing where bandages can actually be used.
Low force high bleed is going to be a fun thing

### CODE REVIEW

**Comment by Niknakflak** [comment]
Stop using the web editor for christ sake

**Comment by PKPenguin321** [comment]
:-1: for the rapier change
:+1: to the refactor

**Comment by Cheridan** [comment]
the greyshirts fear the samurai

**Comment by GunHog** [comment]
Your rapier change was already denied. You are risking a ban by opening more PRs with the same or similar change.

**Comment by jughu** [comment]
RAPIERS CHANGES POEPLE I TELL YA

**Comment by yackemflam** [comment]
@gunhog lot said that the proper solution to this is to make it so that the rapier causes bleeding.

**Comment by PKPenguin321** [comment]
hey wait a second

looking through this all the dismemberment chances you added are abysmal

2% chance for chefs knife?
awful, nevermind :-1:

**Comment by yackemflam** [comment]
Chefs knife couldn't dismember in the first place

File: code/game/objects/items/weapons/melee/energy.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /obj/item/weapon/melee/energy
 	var/active = 0
 	var/force_on = 30 //force when active
+	var/dismemberment_chance_on = 0 //dismemberment chance when active
```
Comment (Jalleo) [inline code/game/objects/items/weapons/melee/energy.dm]: Removing energy sword dismemberment?

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -21,13 +21,14 @@
 		return (OXYLOSS)
 
 
-/obj/item/weapon/melee/sabre
```
Comment (Jalleo) [inline code/game/objects/items/weapons/melee/misc.dm]: I liked the idea of it being a sabre. Also not all sabres were curved.

File: code/game/objects/items/weapons/twohanded.dm [hunk 3]
```diff
@@ -591,14 +617,16 @@
 	w_class = 4
 	slot_flags = SLOT_BACK
 	force_unwielded = 11
-	force_wielded = 20					//I have no idea how to balance
+	force_wielded = 20 //I have no idea how to balance
```
Comment (Jalleo) [inline code/game/objects/items/weapons/twohanded.dm]: Useeless comment from a previous person remove it.

File: code/game/objects/items/weapons/melee/energy.dm (hunk 1 above)
Comment (yackemflam) [inline code/game/objects/items/weapons/melee/energy.dm]: Thanks for the notes, I'll take care of them later.

**Comment by Jalleo** [comment]
To be honest i could comment more but I wont because one. You need to stop using the online editor and push it in more condense comments and two stop trying to push changes that got refused within other changes. You are probably going to get blocked by being considered too much of a nuiscance. 
As in not following standards in other PRs and other stuff.

**Comment by yackemflam** [comment]
Yeah that's why I stopped making commits.

And the rapier change was because lot said that is  a viable way to fix it.

**Comment by Bawhoppen** [comment]
:-1: for incomplete refactor
:+1: for removing dismemberment from rapier

**Comment by KorPhaeron** [comment]
Please learn to use git

**Comment by KorPhaeron** [comment]
We can help you set it up in #coderbus

**Comment by yackemflam** [comment]
Something something git went fucky something something I'm at work currently

**Comment by KorPhaeron** [comment]
well you're definitely not gonna be able to resolve conflicts until you learn to use git, so please let us help you in coderbus

**Pull request closed without merging**